src/ChainlinkOracle.sol

```diff
@@ -35,6 +35,13 @@ contract ChainlinkOracle is IOracle {
 
     /* CONSTRUCTOR */
 
+    /// @dev Here is the list of assumptions on the inputs that guarantees the oracle behaves as expected:
+    /// - Feeds are Chainlink-compliant or the address zero.
+    /// - Feeds have the same behavioral assumptions as Chainlink's.
+    /// - Feeds are set in the correct order.
+    /// - Decimals passed as argument are correct.
```

Comment:
to me it's useless, it's like we can also add "the contract should be deployed"
```suggestion
```

Reply:
I also feel lik this

Reply:
> it's like we can also add "the contract should be deployed"

That's different. it reminds deployers what they should care about those params and it acknowledges to auditors that we don't check it (but we don't forget that this is important for a good setup).

It's closer to your mom saying "you should put your seatbelt". You know that you should do it ofc but forgetting once could kill you.

PS: if we can reduce by 1% the hundreds of issues raised during the competition I would be happy

Reply:
>it reminds deployers what they should care about those params

but it's always like that no? in which contract don't we care about the constructor parameter.

>PS: if we can reduce by 1% the hundreds of issues raised during the competition I would be happy

what report does it prevent (honestly idk)?